telegram/game.py

```diff
@@ -0,0 +1,146 @@
+# it under the terms of the GNU Lesser Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser Public License for more details.
+#
+# You should have received a copy of the GNU Lesser Public License
+# along with this program.  If not, see [http://www.gnu.org/licenses/].
+"""This module contains a object that represents a Telegram Game."""
+
+import sys
+
+from telegram import MessageEntity, TelegramObject, Animation, PhotoSize
+
+
+class Game(TelegramObject):
+    """This object represents a Telegram Game.
+
+    Attributes:
+        title (str): Title of the game.
+        description (str): Description of the game.
+        photo (list[:class:`telegram.PhotoSize`]): List of photos that will be displayed in the
+            game message in chats.
+
+    Keyword Args:
+        text (Optional[str]): Brief description of the game or high scores included in the game
+            message. Can be automatically edited to include current high scores for the game when
+            the bot calls setGameScore, or manually edited using editMessageText.
+            0-4096 characters.
+        text_entities (Optional[list[:class:`telegram.MessageEntity`]]): Special entities that
+            appear in text, such as usernames, URLs, bot commands, etc.
+        animation (Optional[:class:`telegram.Animation`]): Animation that will be displayed in the
+            game message in chats. Upload via BotFather.
+
+    """
+
+    def __init__(self,
+                 title,
+                 description,
+                 photo,
+                 text=None,
+                 text_entities=None,
+                 animation=None,
+                 **kwargs):
+        self.title = title
+        self.description = description
+        self.photo = photo
+        self.text = text
+        self.text_entities = text_entities
+        self.animation = animation
+
+    @staticmethod
+    def de_json(data, bot):
+        """
+        Args:
+            data (dict):
+            bot (telegram.Bot):
+
+        Returns:
+            telegram.Game:
+
+        """
+        if not data:
+            return None
+
+        data['photo'] = PhotoSize.de_list(data.get('photo'), bot)
+        data['text_entities'] = MessageEntity.de_list(data.get('text_entities'), bot)
+        data['animation'] = Animation.de_json(data.get('animation'), bot)
+
+        return Game(**data)
+
+    def to_dict(self):
+        """
+        Returns:
+            dict:
+        """
+        data = super(Game, self).to_dict()
+
+        data['photo'] = [p.to_dict() for p in self.photo]
+        data['text_entities'] = [x.to_dict() for x in self.text_entities]
+
+        return data
+
```

Comment:
@bomjacob 
I think what @jh0ker means is that you don't need to manually serialize a list. the base class will do it for you. (and yes, i missed it myself going over the code)